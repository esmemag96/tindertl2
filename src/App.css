.App {
  text-align: center;
  background-color:white;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

.MuiCardContent-root {
  height: 90%;
}

.MuiCardContent-root>div {
  min-height: 500px !important;
  height: 100% !important;
}

.MuiCardContent-root>div>div {
  max-height: 100% !important;
}

.MuiCardContent-root>div> div > img {
  margin-top: 2%;
  margin-left: 30%;
  max-width: 40% !important;
}