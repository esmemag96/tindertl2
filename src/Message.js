import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import Icon from '@material-ui/core/Icon';
import Avatar from '@material-ui/core/Avatar';
import TextareaAutosize from '@material-ui/core/TextareaAutosize';
import TextField from '@material-ui/core/TextField';
import { fontWeight, width } from '@material-ui/system';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import OneMatch from './OneMatch'
import IconButton from '@material-ui/core/IconButton';
import InputAdornment from '@material-ui/core/InputAdornment';
import MenuItem from '@material-ui/core/MenuItem';

const useStyles = makeStyles(theme => ({
    root: {
        flexGrow: 1,
        color: "black",
        width: "50%",
        height: "100%",
        display: "inline-flex",
        backgroundColor: "white",
        border: "8px solid black",
        borderRadius: "22px",
        fontFamily: "'Poppins', sans-serif",
        minHeight: '100vh',
    },
    iconsStyle: {
        marginTop: "20%",
        fontSize: "30px",
    },
    containerStyle: {
        backgroundColor: "#aa3a2a",
        color: "white",
        marginBottom: "10px",
        width: "100%"
    },
    photo: {
        borderRadius: "50%",
        border: "3px solid black",
        margin: 10,
        width: 60,
        height: 60,
    },
    toMeChatBubble: {
        backgroundColor: "#e8e3e3",
        borderRadius: "15px",
        marginBottom: "10px",
        width: "20%"
    },
    fromMeChatBubble: {
        backgroundColor: "#d68245",
        borderRadius: "15px",
        color: "white",
        marginBottom: "10px",
        width: "20%"
    },
    chatBox: {
        marginTop: "10em",
        marginBottom: "1em"
    },
    textInputStyle: {  
        width:"30em"
    }
}));

export default function Message(props) {
    const classes = useStyles();
    return (
        <div className={classes.root}>
            <Grid container >
                <Grid item xs={3} className={classes.containerStyle}>
                    <Avatar className={classes.photo} src={"https://www.sintesis.mx/wp-content/uploads/2017/09/Indigena-1068x715.jpg"} alt="Photo didn't load :(" />
                </Grid>
                <Grid item xs={9} className={classes.containerStyle}>
                    <Link to="/showprofile" >
                        <h1>Amimitl</h1>
                    </Link>
                </Grid>
                <Grid item xs={12} spacing={10} justify="flex-start" >
                    <h1 className={classes.toMeChatBubble}>hola</h1>
                </Grid>
                <Grid container direction="column" justify="center" alignItems="flex-end" item xs={12} spacing={10} >
                    <h1 className={classes.fromMeChatBubble}>hola</h1>
                </Grid>
                <Grid item xs={12} className={classes.chatBox} >
                    <TextField
                        id="outlined-adornment-password"
                        className={classes.textInputStyle}
                        variant="outlined"
                        type=""
                        label="Send message"
                        value=""
                        onChange=""
                        InputProps={{
                            endAdornment: (
                                <InputAdornment position="end">
                                    <Icon
                                        edge="end"
                                        aria-label="toggle password visibility"
                                        onClick=""
                                        onMouseDown=""
                                    >
                                    send
                                    </Icon>
                                </InputAdornment>
                            ),
                        }}
                    />
                </Grid>
            </Grid>
        </div>
    );
}
