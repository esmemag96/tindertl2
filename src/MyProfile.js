import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import Fab from '@material-ui/core/Fab';
import Avatar from '@material-ui/core/Avatar';
import Checkbox from '@material-ui/core/Checkbox';
import FormControlLabel from '@material-ui/core/FormControlLabel';

import { BrowserRouter as Router, Route, Link } from "react-router-dom";

const useStyles = makeStyles(theme => ({
    root: {
        backgroundColor: "#aa3a2a",
        fontFamily: "'Poppins', sans-serif",
        color: "white"
    },
    containerStyle: {
        width: "50%",
        display: "inline-flex",
        backgroundColor: "aa3a2a",
    },
    photo: {
        margin: 10,
        width: 300,
        height: 300,
    },
    fab: {
        margin: theme.spacing(1),
        backgroundColor: "#009688",
        color: "white"
    },
    fab1: {
        margin: theme.spacing(1),
        backgroundColor: "#009688",
        color: "white",
        marginTop: "-5em",
        marginLeft: "11em"
    },
    formControl: {
        width: "30em",
        color: "white"
    },
    checkboxStyle: {
        color: '#009688 !important'
    }
}));

export default function ShowPhoto() {
    const [state, setState] = React.useState({
        checkedA: true,
        checkedB: true,
        checkedC: true,
        checkedD: true,
    });
    const handleChange = name => event => {
        setState({ ...state, [name]: event.target.checked });
    };
    const classes = useStyles();
    return (
        <div className={classes.root} >
            <Grid container className={classes.containerStyle} justify="center">
                <Grid item md={6} xs={12}>
                    <Avatar src={'https://fuckingtonpost.files.wordpress.com/2013/08/323-brasil-la-propuesta-del-congreso-serc3ada-un-desastre-absoluto-para-los-indc3adgenas.jpg'} className={classes.photo} />
                </Grid>
                <Grid item xs={12}>
                    <Fab aria-label="add" className={classes.fab1}>
                        <i class="material-icons">insert_photo</i>
                    </Fab>
                </Grid>
                <Grid item xs={12}>
                    <h1 className={classes.name}>Atzin, Jilotepec</h1>
                </Grid>
                <Grid item xs={12}>
                    <h4 className={classes.bio}> Hablo mixteco, Otomi y Nahuatl, busco amigos para la peda y el amor</h4>
                </Grid>
                <Grid item xs={3}>
                Editar
                    <Fab aria-label="add" className={classes.fab}>
                       <i class="material-icons">edit</i>
                    </Fab>
                </Grid>
                <Grid item xs={12}>
                    <h4 className={classes.name}>Quiero conocer gente de:</h4>
                    <FormControlLabel
                        control={
                            <Checkbox
                                className={classes.checkboxStyle}
                                checked={state.checkedA}
                                onChange={handleChange('checkedA')}
                                value="checkedB"
                            />
                        }
                        label="Nahuas"
                    />
                    <FormControlLabel
                        control={
                            <Checkbox
                                className={classes.checkboxStyle}
                                checked={state.checkedB}
                                onChange={handleChange('checkedB')}
                                value="checkedB"
                            />
                        }
                        label="Mayas"
                    />
                    <FormControlLabel
                        control={
                            <Checkbox
                                className={classes.checkboxStyle} 
                                checked={state.checkedC}
                                onChange={handleChange('checkedC')}
                                value="checkedB"
                            />
                        }
                        label="Zapotecas"
                    />
                    <FormControlLabel
                        control={
                            <Checkbox
                                className={classes.checkboxStyle}
                                checked={state.checkedD}
                                onChange={handleChange('checkedD')}
                                value="checkedB"
                            />
                        }
                        label="Mixtecas"
                    />
                </Grid>
            </Grid>
        </div>
    );
}