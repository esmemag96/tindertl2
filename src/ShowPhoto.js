import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import Fab from '@material-ui/core/Fab';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";

const useStyles = makeStyles(theme => ({
  root: {
    flexGrow: 1,
    fontFamily: "'Poppins', sans-serif",
    minHeight: '100vh',
  },
  containerStyle: {
    color: "black",
    width: "50%",
    display: "inline-flex",
    backgroundColor: "white",
    border: "8px solid black",
    borderRadius: "22px",
    fontFamily: "'Poppins', sans-serif",
  },
  fab: {
    margin: theme.spacing(1),
    backgroundColor: "#009688",
    color:"white"
  },
  photo: {
      width:"100%"
  }
}));

export default function ShowPhoto() {
  const classes = useStyles();
  return(
    <div className={classes.root}>
      <Grid container className={classes.containerStyle}>
        <Grid item xs={12}>
          <h1 className={classes.name}>Yalitza</h1>
        </Grid>
        <Grid item xs={12}>
          <img className={classes.photo} src={'https://img-s-msn-com.akamaized.net/tenant/amp/entityid/AAFwp4s.img?h=375&w=624&m=6&q=60&o=f&l=f&x=573&y=237'} alt="Photo didn't load :("></img>
        </Grid>
        <Grid item xs={12}>
          <h4 className={classes.bio}> Actriz de cine y maestra mexicana, reconocida por mi participaci√≥n en la cinta Roma, no busco nada serio, no mando nudes</h4>
        </Grid>
        <Grid item xs={6}>
            <Fab aria-label="add" className={classes.fab}>
                <i class="material-icons">thumb_down</i>
            </Fab>
        </Grid>
        <Grid item xs={6}>
            <Fab aria-label="add" className={classes.fab}>
                <i class="material-icons">thumb_up</i>
            </Fab>
        </Grid>
      </Grid>
    </div>
  );
}