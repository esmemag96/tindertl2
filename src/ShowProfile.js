import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import Fab from '@material-ui/core/Fab';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";

const useStyles = makeStyles(theme => ({
    containerStyle: {
        color: "black",
        width: "50%",
        display: "inline-flex",
        backgroundColor: "white",
        border: "8px solid black",
        borderRadius: "22px",
        fontFamily: "'Poppins', sans-serif",
        minHeight: '100vh',
    },
    fab: {
        margin: theme.spacing(1),
        backgroundColor: "#b3150c",
        color: "white"
    },
    photo: {
        width: "100%"
    }
}));

export default function ShowPhoto() {
    const classes = useStyles();
    return (
        <div>
            <Grid container className={classes.containerStyle}>
                <Grid item xs={12}>
                    <h1 className={classes.name}> Amimitl, Tepoztl√°n</h1>
                </Grid>
                <Grid item xs={12}>
                    <img className={classes.photo} src={'https://www.sintesis.mx/wp-content/uploads/2017/09/Indigena-1068x715.jpg'} alt="Photo didn't load :("></img>
                </Grid>
                <Grid item xs={12}>
                    <h4 className={classes.bio}> Soy una chica sencilla, me gusta pasear y comer </h4>
                </Grid>
                <Grid item xs={12}>
                    Unmatch
            <Fab aria-label="add" className={classes.fab}>
                        <i class="material-icons">cancel</i>
                    </Fab>
                </Grid>
            </Grid>
        </div>
    );
}