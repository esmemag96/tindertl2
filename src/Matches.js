import React from 'react';
import logo from './images/tindertlFull.png';
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import Icon from '@material-ui/core/Icon';
import { fontWeight } from '@material-ui/system';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import OneMatch from './OneMatch'
import Message from './Message'

const useStyles = makeStyles(theme => ({
    root: {
        flexGrow: 1,
        fontFamily: "'Poppins', sans-serif",
        minHeight: '100vh',
    },
    containerStyle: {
        color: "black",
        width: "70%",
        height: "100%",
        display: "inline-flex",
        backgroundColor: "white",
        border: "8px solid black",
        borderRadius: "22px"
    },
}));

export default function Matches() {
    const classes = useStyles();
    return (
        <div className={classes.root}>
            <Grid container className={classes.containerStyle}>
                <Grid item xs={12}>
                    <h1>Matches</h1>
                    <Link to="/message/001">
                        <OneMatch photo="https://www.gob.mx/cms/uploads/image/file/317714/MUJERES_IND_GENAS__5_.jpg" name="Amimitl" id="001" />
                    </Link>
                    <OneMatch photo="https://www.gob.mx/cms/uploads/image/file/317714/MUJERES_IND_GENAS__5_.jpg" name="Xochitl" id="002" />
                    <OneMatch photo="https://i0.wp.com/mujeresconfiar.com/wp-content/uploads/2018/09/Hilda-Domico-Bailarin-1.jpg?fit=1024%2C702&ssl=1" name="Yanetzy" id="003" />
                </Grid>
            </Grid>
        </div>
    );
}
