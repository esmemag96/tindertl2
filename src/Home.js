import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import Fab from '@material-ui/core/Fab';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import ShowPhoto from './ShowPhoto'

const useStyles = makeStyles(theme => ({
  root: {
    flexGrow: 1,
    backgroundColor: "#aa3a2a",
    fontFamily: "'Poppins', sans-serif"
  },
}));

export default function Home() {
  const classes = useStyles();
  return(
    <div className={classes.root}>
        <ShowPhoto/>
    </div>
  );
}