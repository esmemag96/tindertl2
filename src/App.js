import React from 'react';
import './App.css';
import AppHeader from './appHeader'
import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";
import { makeStyles } from '@material-ui/core/styles';
//Components
import Matches from './Matches'
import Home from './Home'
import Message from './Message'

const useStyles = makeStyles(theme => ({
  root: {
    flexGrow: 1,
    backgroundColor: "#aa3a2a",
    fontFamily: "'Poppins', sans-serif"
  },
}));

function App() {
  const classes = useStyles();
  return (
    <div className="App">
      <Router>
        <AppHeader />
        <Switch>
          <Route exact path="/" component={Home} />
          <Route path="/matches" component={Matches} />
          <Route path="/message/:id" component={Message} />
        </Switch>
      </Router>
    </div>
  );
}

export default App;
