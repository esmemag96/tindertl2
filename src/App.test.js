import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import AppHeader from './appHeader'
import AppFooter from './appFooter'
import Matches from './Matches'
import Home from './Home'
import Message from './Message'
import MyProfile from './MyProfile'
import ShowProfile from './ShowProfile'
import ShowPhoto from './ShowPhoto'
import "@testing-library/jest-dom/extend-expect"
import { render, cleanup } from "@testing-library/react"

import renderer from "react-test-renderer"

afterEach(cleanup)
/*
it('Home renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Home />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Matches renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Matches />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Message renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Message />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('MyProfile renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<MyProfile />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('ShowProfile renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ShowProfile />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('App renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});
*/

it("Functional test passed", () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("Functional test passed", () => {
  const tree = renderer.create(<Matches />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("Functional test passed", () => {
  const tree = renderer.create(<Message />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("Functional test passed", () => {
  const tree = renderer.create(<MyProfile />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("Functional test passed", () => {
  const tree = renderer.create(<ShowProfile />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("Functional test passed", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
})
