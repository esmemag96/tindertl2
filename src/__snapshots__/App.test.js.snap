// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Functional test passed 1`] = `
<div
  className="makeStyles-root-197"
>
  <div
    className="MuiGrid-root makeStyles-containerStyle-198 MuiGrid-container MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
    >
      <div
        className="MuiAvatar-root makeStyles-photo-199"
      >
        <img
          className="MuiAvatar-img"
          src="https://fuckingtonpost.files.wordpress.com/2013/08/323-brasil-la-propuesta-del-congreso-serc3ada-un-desastre-absoluto-para-los-indc3adgenas.jpg"
        />
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <button
        aria-label="add"
        className="MuiButtonBase-root MuiFab-root makeStyles-fab1-201"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiFab-label"
        >
          <i
            class="material-icons"
          >
            insert_photo
          </i>
        </span>
      </button>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h1>
        Atzin, Jilotepec
      </h1>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h4>
         Hablo mixteco, Otomi y Nahuatl, busco amigos para la peda y el amor
      </h4>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
    >
      Editar
      <button
        aria-label="add"
        className="MuiButtonBase-root MuiFab-root makeStyles-fab-200"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiFab-label"
        >
          <i
            class="material-icons"
          >
            edit
          </i>
        </span>
      </button>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h4>
        Quiero conocer gente de:
      </h4>
      <label
        className="MuiFormControlLabel-root"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-229 MuiCheckbox-root MuiCheckbox-colorSecondary makeStyles-checkboxStyle-203 PrivateSwitchBase-checked-230 Mui-checked MuiIconButton-colorSecondary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <input
              checked={true}
              className="PrivateSwitchBase-input-232"
              data-indeterminate={false}
              onChange={[Function]}
              type="checkbox"
              value="checkedB"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
              />
            </svg>
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Nahuas
        </span>
      </label>
      <label
        className="MuiFormControlLabel-root"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-229 MuiCheckbox-root MuiCheckbox-colorSecondary makeStyles-checkboxStyle-203 PrivateSwitchBase-checked-230 Mui-checked MuiIconButton-colorSecondary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <input
              checked={true}
              className="PrivateSwitchBase-input-232"
              data-indeterminate={false}
              onChange={[Function]}
              type="checkbox"
              value="checkedB"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
              />
            </svg>
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Mayas
        </span>
      </label>
      <label
        className="MuiFormControlLabel-root"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-229 MuiCheckbox-root MuiCheckbox-colorSecondary makeStyles-checkboxStyle-203 PrivateSwitchBase-checked-230 Mui-checked MuiIconButton-colorSecondary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <input
              checked={true}
              className="PrivateSwitchBase-input-232"
              data-indeterminate={false}
              onChange={[Function]}
              type="checkbox"
              value="checkedB"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
              />
            </svg>
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Zapotecas
        </span>
      </label>
      <label
        className="MuiFormControlLabel-root"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-229 MuiCheckbox-root MuiCheckbox-colorSecondary makeStyles-checkboxStyle-203 PrivateSwitchBase-checked-230 Mui-checked MuiIconButton-colorSecondary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <input
              checked={true}
              className="PrivateSwitchBase-input-232"
              data-indeterminate={false}
              onChange={[Function]}
              type="checkbox"
              value="checkedB"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
              />
            </svg>
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Mixtecas
        </span>
      </label>
    </div>
    <div
      className="Charts"
    >
      <div
        className="rv-xy-plot "
        style={
          Object {
            "height": "300px",
            "width": "300px",
          }
        }
      >
        <svg
          className="rv-xy-plot__inner"
          height={300}
          onClick={[Function]}
          onDoubleClick={[Function]}
          onMouseDown={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseMove={[Function]}
          onMouseUp={[Function]}
          onTouchCancel={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          onWheel={[Function]}
          width={300}
        >
          <g
            className="rv-xy-plot__series rv-xy-plot__series--bar "
            transform="translate(40,10)"
          >
            <rect
              height={222.2222222222222}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={1.875}
              y={27.7777777777778}
            />
            <rect
              height={138.88888888888889}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={26.875}
              y={111.11111111111111}
            />
            <rect
              height={111.11111111111109}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={51.875}
              y={138.8888888888889}
            />
            <rect
              height={250}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={76.875}
              y={0}
            />
            <rect
              height={27.77777777777777}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={101.875}
              y={222.22222222222223}
            />
            <rect
              height={194.44444444444446}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={126.875}
              y={55.55555555555554}
            />
            <rect
              height={166.66666666666666}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={151.875}
              y={83.33333333333334}
            />
            <rect
              height={83.33333333333331}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={176.875}
              y={166.66666666666669}
            />
            <rect
              height={55.55555555555554}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={201.875}
              y={194.44444444444446}
            />
            <rect
              height={0}
              onClick={[Function]}
              onContextMenu={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "fill": "#12939A",
                  "opacity": 1,
                  "stroke": "#12939A",
                }
              }
              width={21.25}
              x={226.875}
              y={250}
            />
          </g>
        </svg>
      </div>
    </div>
  </div>
</div>
`;

exports[`Functional test passed 2`] = `
<div>
  <div
    className="MuiGrid-root makeStyles-containerStyle-291 MuiGrid-container"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h1>
         Amimitl, Tepoztlán
      </h1>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <img
        alt="Photo didn't load :("
        className="makeStyles-photo-293"
        src="https://www.sintesis.mx/wp-content/uploads/2017/09/Indigena-1068x715.jpg"
      />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h4>
         Soy una chica sencilla, me gusta pasear y comer 
      </h4>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      Unmatch
      <button
        aria-label="add"
        className="MuiButtonBase-root MuiFab-root makeStyles-fab-292"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiFab-label"
        >
          <i
            class="material-icons"
          >
            cancel
          </i>
        </span>
      </button>
    </div>
  </div>
</div>
`;
